distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc

```diff
@@ -0,0 +1,15 @@
+:title: Attributes Added by Pre-Ingest Plugins
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How pre-ingest plugins add attributes to metacards.
+:order: 05
+
+=== {title}
+
+The <<_pre_ingest_plugins,Pre-Ingest Plugins>> are responsible for setting attribute fields on resulting metacards after the document has been transformed.
+
+* The <<_expiration_date_pre_ingest_plugin,Expiration Date Pre-Ingest Plugin>> adds or updates expiration dates which can be used later for archiving old data.
+* The <<_geocoder_plugin,Geocoder Plugin>> is responsible for populating the metacard’s `Location.COUNTRY_CODE` attribute if the metacard has an associated location. If the metacard’s country code is already populated, the plugin will not override it.
```

Comment:
❓ Is the `location` here the metacard attribute `location` or does it cover other attributes?

Reply:
Looks like it is just the `location.COUNTRY_CODE` attribute that gets modified.